packages/longhorn/generated-changes/patch/questions.yml.patch

```diff
@@ -36,7 +36,12 @@
      description: "Specify Longhorn Instance Manager Image Repository"
      type: string
      label: Longhorn Instance Manager Image Repository
-@@ -59,7 +59,7 @@
+     group: "Longhorn Images Settings"
+   - variable: image.longhorn.instanceManager.tag
+-    default: v1_20201216
++    default: v1_20210621
```

Comment:
What is this varaible?

Reply:
Oh. I just realized the original Chart is not up-to-date. Thank you for pointing it out!